docs/Guides/Automate/WebhookAction.md

```diff
@@ -0,0 +1,167 @@
+# Use the trigger a webhook action
+
+You can take action in your systems based on events in the Vendasta platform by using automations. With any automation you can add the `Trigger a webhook` step to be notified when something happens. 
+
+## Overview
+
+In this guide we will create an automation, listen for the event and retrieve details of the affected records.
+
+## Prerequisites
+
+This guide assumes you are familiar with creating automations within the platform. If you are not familiar you should [learn more about automations](https://support.vendasta.com/hc/en-us/sections/4406950706583-Automations) first.
+
+
+## Step 1: Automation setup
+
+In Partner Center go to the automations screen and create an automation based on the trigger and criteria that you are interested in.
+
+Add the `Trigger a webhook` step.
+
+![](webhookAddPart1.png)
+
+The **webhook URL** will be an endpoint that you have built in step 2. It will likely be one of your web servers but could also be a 3rd party system like Zapier. 
+
+For testing purposes [webhook.site](https://webhook.site/) is a great tool for viewing what is sent. You may also use a tool like [ngrok](https://ngrok.com/) to route requests to your local development computer.
+
+The **Verifier Token** is something that you can makeup. We will send it along with the webhook to help you verify that the request actually came from us.
```

Comment:
Maybe a bit more explanation here. Maybe something like The Verifier Token is named by you and is sent along with the webhook to help you verify that the request actually came from us.

Reply:
how about
The Verifier Token is designated by you and is sent along with the webhook to help you verify that the request originated from this automation.

Reply:
That's perfect!